src/main/resources/db/migration/V2__create_users_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS `Users` (
```

Comment:
do we need a hash or salt stored to decrypt the password?

Reply:
As long as we use Argon2 we can just do `Argon2PasswordEncoder.matches(<plaintext password from input>, <encoded password in DB>)` and this'll save us decrypting it unnecessarily to verify if they match or not.
Is there a use case I might be missing where the password retrieved from the DB would ever need to return to plaintext?